### PR TITLE
removed brackets

### PR DESCRIPTION
**Pull request by RAJU2529**
as per user feedback  #7179  so i  removed brackets.

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/admin/manage/manage-addins-in-the-admin-center.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ description: "Learn about using Centralized add-ins to deploy add-ins to users a
 
 # Manage add-ins in the admin center
 
-Office add-ins help you personalize your documents and streamline the way you access information on the web (see [Start using your Office add-in](https://support.microsoft.com/office/82e665c4-6700-4b56-a3f3-ef5441996862)). 
+Office add-ins help you personalize your documents and streamline the way you access information on the web see [Start using your Office add-in](https://support.microsoft.com/office/82e665c4-6700-4b56-a3f3-ef5441996862).
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/manage/manage-addins-in-the-admin-center.md]:
```suggestion
Office add-ins help you personalize your documents and streamline the way you access information on the web. See [Start using your Office add-in](https://support.microsoft.com/office/82e665c4-6700-4b56-a3f3-ef5441996862). 
```

File: microsoft-365/admin/manage/manage-addins-in-the-admin-center.md [hunk 2]
```diff
@@ -125,7 +125,7 @@ To prevent a user from signing in with a Microsoft account, you can restrict log
 
 ## More about the end-user experience with add-ins
 
-After you deploy an add-in, your end users can start using it in their Office applications (see [Start using your Office Add-in](https://support.microsoft.com/office/82e665c4-6700-4b56-a3f3-ef5441996862)). The add-in appears on all platforms that the add-in supports.
+After you deploy an add-in, your end users can start using it in their Office applications see [Start using your Office Add-in](https://support.microsoft.com/office/82e665c4-6700-4b56-a3f3-ef5441996862). The add-in appears on all platforms that the add-in supports.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/manage/manage-addins-in-the-admin-center.md]:
```suggestion
After you deploy an add-in, your end users can start using it in their Office applications. The add-in appears on all platforms that the add-in supports. See [Start using your Office Add-in](https://support.microsoft.com/office/82e665c4-6700-4b56-a3f3-ef5441996862). 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kwekuako Please merge this PR. Thanks.

**Pull request merged**